server.R

```diff
@@ -112,8 +112,9 @@ server <- function(input, output) {
   )
   
   
+  # no longer need for this
   # mapa RO
-  rondonia <- jsonlite::read_json("https://raw.githubusercontent.com/luizpedone/municipal-brazilian-geodata/master/data/RO.json")
+  #rondonia <- jsonlite::read_json("https://raw.githubusercontent.com/luizpedone/municipal-brazilian-geodata/master/data/RO.json")
   
   rondonia$features <- rondonia$features %>%
     purrr::map(function(x){

```

Comment:
@JohnCoene  
The e_map_register_ui(...) has generated the map but without tooltip like before.
How can I adjust the tooltip without object 'rondonia'?

Reply:
Good point, should work with [nameProperty](https://echarts.apache.org/en/option.html#series-map.nameProperty) in `e_map`

```r
e_map(Casos, map = "Rondonia", emphasis = list(itemStyle = list(shadowBlur = 3)),
            roam = TRUE, zoom = 1, nameProperty = "NOME")
```

Reply:
There don't need these changes:

`rondonia$features <- rondonia$features %>%
    purrr::map(function(x){ 
            x$properties$name <- x$properties$NOME             # copy NOME to name for tooltip
      return(x)
    })`

Reply:
`And about these changes?

      rondonia$features <- rondonia$features %>% purrr::map(function(x){
              x$properties$name <- x$properties$NOME
      return(x)
    })
`    

Only 'nameProperty = "NOME" don't make any change on the map.